05-injection-part-2/README.md

```diff
@@ -128,7 +129,7 @@ This is not a code injection vulnerability as each website is vulnerable to this
 ![Self XSS](https://github.com/hexcellents/sss-web/blob/master/05-injection-2/support/self-xss.png)
 
 ### Mutated XSS
-Mutated XSS happens when the browser tries to fix and rewrite invaild HTML but fails doing so thus executing attacker's code. Because it depends from browser to browser, is extremely hard to detect or sanitize within the websites application logic. At the end of this page you will find a video explaining a mXSS on Google Search.
+Mutated XSS happens when the browser tries to fix and rewrite invaild HTML but fails doing so thus executing attacker's code. Because it depends from browser to browser, is extremely hard to detect or sanitize within the websites application logic. At the end of this page you will find a video explaining a mXSS on Google Search, and also check the link for one of the first papers describing mXSS attacks.
 
 ## Tasks
```

Comment:
I think it should be Challenges, not Tasks.
Also, I saw that you used "Tasks" in multiple places, is there a reason behind it?(Line 242)

Reply:
"Tasks" were used before, I didn't kept them. They were never changed. I will push required modifications.